test/e2e/network/networking.go

```diff
@@ -327,6 +328,36 @@ var _ = common.SIGDescribe("Networking", func() {
 
 		ginkgo.It("should update endpoints: http", func(ctx context.Context) {
 			config := e2enetwork.NewNetworkingTestConfig(ctx, f)
+
+			// start of intermittent code snippet to understand the reason for flaky behaviour
+			// TODO @aroradaman @aojea remove this once issue #123760 is resolved
+			// streaming logs for netserver-0 which will be deleted during the test
+			// (ref: https://github.com/kubernetes/kubernetes/issues/123760)
+			go func(pod *v1.Pod) {
```

Comment:
is not simpler to capture the pod name instead of passing the pointer? 

`pod0name = config.EndpointPods[0].Name`

we also don't need to pass the parameters if we capture the value before the goroutine